src/components/home/Banners.jsx

```diff
@@ -0,0 +1,14 @@
+export default function Banner({ banners }) {
```

Comment:
Doing the slider manually takes a lot of time. However, using the carousel of react bootstrap could make the development much simpler. Here is the link [React Bootstrap Carousel](https://react-bootstrap.github.io/components/carousel/)